src/content/reference/react/useSyncExternalStore.md

```diff
@@ -377,18 +377,18 @@ Make sure that `getServerSnapshot` returns the same exact data on the initial cl
 
 This error means your `getSnapshot` function returns a new object every time it's called, for example:
 
-```js {2-5}
+```js {3-5}
 function getSnapshot() {
-  // 🔴 Do not return always different objects from getSnapshot
+  // 🔴 This creates a new object every time getSnapshot is called, even when myStore.todos has not changed
```

Comment:
This is more clear, but we try to keep the line length smaller so you don't need to scroll like this on smaller screen sizes:

<img width="775" alt="Screenshot 2024-01-05 at 5 27 56 PM" src="https://github.com/reactjs/react.dev/assets/2440089/29326d7a-e4c5-48bc-92b5-dbda07c8c3cb">

wdyt about making it two lines?